api.js

```diff
@@ -24,16 +24,21 @@ function Api(options) {
 	EventEmitter.call(this);
 
 	this.options = objectAssign({
-		cwd: process.cwd(),
 		resolveTestsFrom: process.cwd(),
 		match: []
 	}, options);
 
 	this.options.require = (this.options.require || []).map(function (moduleId) {
+		var originalCwd = process.cwd();
+		if (options.pkgDir) {
+			process.chdir(options.pkgDir);
```

Comment:
Using `process.chdir` is almost always a bad idea. It changes the CWD globally and can cause race issues and all kinds of problems.